cosign.yaml

```diff
@@ -15,17 +15,20 @@ environment:
       - busybox
       - ca-certificates-bundle
       - go
+  environment:
+    CGO_ENABLED: "0"
 
 pipeline:
   - uses: fetch
     with:
       uri: https://github.com/sigstore/cosign/archive/v${{package.version}}/cosign-v${{package.version}}.tar.gz
       expected-sha256: 5d4950f8718c231fb393206bc9e5b2b49be0eb0d3204563744b3c80a7526764b
 
-  - runs: |
-      CGO_ENABLED=0 go build \
-        -trimpath -ldflags "-buildid= -X sigs.k8s.io/release-utils/version.gitVersion=${{package.version}}" \
-        -o "${{targets.destdir}}/usr/bin/cosign" ./cmd/cosign
+  - uses: go/build
+    with:
+      packages: ./cmd/cosign
+      output: cosign
+      ldflags: -X sigs.k8s.io/release-utils/version.gitVersion=${{package.version}}
```

Comment:
Just checking does variable expansion work as we expect here? I don't think cosign would complain if it was told its git version was the string `${{package.version}}`.

Reply:
Yep, I tested that!